docs/architecture/adr-044-lite-client-with-weak-subjectivity.md

```diff
@@ -0,0 +1,70 @@
+# Tendermint Lite Client with Weak Subjectivity ADR
+
+Changelog:
+- 13-07-2019: Initial Draft
+## Context
+The concept of light clients was introduced in the Bitcoin white paper. It describes a watcher of distributed consensus process that only validates the consensus algorithm and not the state machine transactions within.
```

Comment:
Two things to note:
- Light clients are expected to agree with full nodes on the canonical chain
- Tendermint provides a somewhat different (stronger) light client model under eclipse, since the eclipsing node(s) can only fool the light client if they have two-thirds of the private keys from the last root of trust.